### ISSUE TITLE
Electrum History Page Issues

### ISSUE THREAD

**Issue by spbubble123** [chris-belcher/electrum-personal-server#232]
Sorry if this is already resolved, but I'm just starting out with EPS.

After setting up and running EPS, when I start Electrum, the History tab on Electrum doesn't look right.

Linking to one of the standard servers, and an existing Electrum watching wallet the History page shows the number of confirmations, and the date, as below:

![Network servers](https://user-images.githubusercontent.com/30923584/104488057-c2336b00-55c5-11eb-80b7-7c88174018bd.jpg)

However, using EPS shows no confirmations and no dates, as in the following image.

![EPS](https://user-images.githubusercontent.com/30923584/104488069-c790b580-55c5-11eb-8dfb-62d33a4da85b.jpg)

What am I doing wrong?

**Comment by chris-belcher** [comment]
Did you rescan?

**Comment by spbubble123** [comment]
I did, but I'll rescan once more....

**Comment by spbubble123** [comment]
It's now OK. 

User ignorance! I had not really understood the instructions on your page.

I needed to make sure that Bitcoin Core was running the wallet named in the config.ini file; ie electrumpersonalserver, and not the default bitcoin core wallet. It's obvious really :-)

The rescan then populated the bitcoin core wallet, which showed the transactions in the electrum wallet.

When I ran the electrum-personal-server with the config.ini file, it connected to the bitcoin core wallet, and picked up the confirmations and dates.

Clearly, the sample config.ini identifies the steps required, but the context ie the bitcoin core wallet, was not obvious to a novice like me.

Thanks for your patience.
